scripts/travis/build-all-in-one-image.sh

```diff
@@ -2,7 +2,7 @@
 
 set -exu
 
-BRANCH=${BRANCH:?'missing BRANCH env var'}
+BRANCH=${BRANCH_NAME:?'missing BRANCH env var'}
```

Comment:
can we rename this back to minimize changes?

Reply:
Agree, sure